### DISCUSSION TITLE
Is either Chromium or Firefox the "suggested browser"

### DISCUSSION THREAD

**Discussion by GNOMES** [darkreader/darkreader#9299] [General]
I am reconsidering my browser of choice lately (Brave vs FF), and plugin support/performance are my biggest sway factors.

Does DarkReader development/performance favor one browser or another?

Ublock Origin for example is better on Firefox, and soon to be limited on Chromium due to Manifest v3 change (I know Dark Reader is not impacted after recent blog post. As such the Dev is pretty open that Firefox is the suggested browser.

**Comment by Gusted** [comment]
> Does DarkReader development/performance favor one browser or another?

I personally use Firefox and therefor while developing Dark Reader I always factor in if Firefox and Chromium has support for X or Y. For performance 🤷🏽‍♀️, YMMV but performance should be great on Firefox and Chromium and if there is a performance bug, Firefox is the golden standard to investigate such bug. Unless you're benchmarking, you won't notice.



> Ublock Origin for example is better on Firefox, and soon to be limited on Chromium due to Manifest v3 change (I know Dark Reader is not impacted after recent blog post. As such the Dev is pretty open that Firefox is the suggested browser.

Which blog post? Dark Reader will be impacted, however not in the same way as Ublock Origin does. Most of our main architecture will still work fine, however certain code that relied on some sketchy details will be really rough to be MV3-compatible.

**Reply by GNOMES** [reply]
I saw this post https://darkreader.org/blog/5m-users/ about v3 working/work is progressing, so I gathered that there was no problems.
